licenses.json

```diff
@@ -1,4 +1,5 @@
 {
 	"GPL-3.0": "GPLv3",
-	"GPL-3.0+": "GPLv3+"
+	"GPL-3.0+": "GPLv3+",
+  "GPLv3-or-later": "GPLv3+"
```

Comment:
Technically this is an invalid SPDX code so the source package should receive a patch. The correct version is `GPL-3.0-or-later` which would be `GPLv3+` in Fedora terms. It would also be nice to keep the indenting consistent.

Reply:
Where did you find this particular variant? It shouldn't exist at all because it's a combination of two styles.

Reply:
I wish I could remember. I'd have to regenerate them all to find it.